errata_tool/erratum.py

```diff
@@ -722,13 +722,14 @@ def setFileInfo(self, file_info):
         self._processResponse(r)
 
     def removeBuilds(self, buildlist):
-        if type(buildlist) is not str and type(buildlist) is not list:
+        """Remove build(s) from advisory"""
+        if not isinstance(buildlist, (str, list)):
             raise IndexError
 
         # Removing builds
         # REFERENCE
 
-        if isinstance(buildlist, six.string_types):
```

Comment:
hmm not intended to remove the six.string_types from this one, as that was here before and dealt with unicode strings in py2.

Reply:
it was my code change and I need to revert this change.

Reply:
done